README.md

```diff
@@ -145,9 +147,11 @@ A la suite de cette ligne, vous pouvez voir l'objet qui représente la curiosit
 
 La ligne avec l'identifiant 1 de la table ````Curiosity```` de la base de données a été abstraite dans le modèle (ou classe) ````Curiosity```` de l'application, ce qui donne une instance de ce modèle. 
 
+Nous allons maintenant quitter la console Rails : pour cela, tapez "exit" dans la console à partir du terminal. 
+
 #### Ajouter de nouveaux attributs
 
-Nous aimerions que chaque curiosité ait une catégorie spécifique. Par exemple:
+Nous aimerions que chaque curiosité aie une catégorie spécifique. Par exemple:
```

Comment:
que chaque curiosité ait => subjonctif présent 
I'd like to keep to old version :) Could you remove this change ? Thanks !